client/src/planwise/client/projects2/components/settings.cljs

```diff
@@ -232,20 +232,31 @@
      [:label "Tags: " [tag-set @tags read-only]]
      [count-providers @tags @current-project provider-unit]]))
 
+; Accept previous coverage algorithms as the new one combining both
+(defn- get-coverage-algorithm
+  [coverage]
+  (case coverage
+    "driving-friction" "drive-walk-friction"
+    "walking-friction" "drive-walk-friction"
+    coverage))
```

Comment:
I think we should do this migration server side given that the list of supported algorithms now only returns `drive-walk-friction`, eg. in `planwise.component.projects2/get-project`. You _could_ probably remove the old cases of `compute-coverage-polygon` in that case as well.

Reply:
I'll do it.